.github/workflows/test.yaml

```diff
@@ -82,18 +80,16 @@ jobs:
           ~/.vvm
         key: compiler-cache
 
-    - name: Cache compiler installations
+    - name: Cache Brownie Build Folders
       uses: actions/cache@v2
       with:
         path: |
-          ~/.solcx
-          ~/.vvm
-        key: ${{ runner.os }}-compiler-cache
+          ~/.brownie
+          ./build
+        key: build-cache
 
     - name: Setup node.js
       uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
```

Comment:
why did you remove this? I don't know what's the default, but let's have a hardcoded version at least so it's reproducible.

Reply:
That's not even a hardcoded version, it just uses 14.x by default